homeassistant/components/homematicip_cloud/cover.py

```diff
@@ -48,22 +48,22 @@ def current_cover_position(self):
         """Move the cover to a specific position."""
         position = kwargs[ATTR_POSITION]
         level = position / 100.0
-        await self._device.set_shutter_level(level)
+        await self._device.set_shutter_level(1-level)
 
     @property
     def is_closed(self):
         """Return if the cover is closed."""
         if self._device.shutterLevel is not None:
-            return self._device.shutterLevel == 0
+            return self._device.shutterLevel == 1
```

Comment:
Let's define 2 constants for open and closed.  This will reduce confusion in the future!